ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main__gcp-4.17-nightly-x86.yaml

```diff
@@ -44,6 +44,7 @@ tests:
   timeout: 6h0m0s
 - always_run: false
   as: node-density-heavy-etcd-fips-24nodes
+  cron: 0 11 5 * *
```

Comment:
If you're open to this running on other days, placing it on day 12 or 26 instead of 5 will balance more. Otherwise, we'll end up with 16 runs on the 5th, and 11 runs on the 26th.